isos/bootstrap/bootstrap

```diff
@@ -38,6 +38,9 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     mount -t tmpfs -o size=64m tmpfs ${MOUNTPOINT}/.tether/
 
     # enable full system functionality in the container
+    ln -s lib64 ${MOUNTPOINT}/.tether/lib
+    mkdir -p ${MOUNTPOINT}/.tether/{lib64,usr/lib/iptables}
```

Comment:
@mhagen-vmware when you create symbolic links by `ln -s`, the target file does not have to exist beforehand.